testing/src/org/immutables/check/IterableChecker.java

```diff
@@ -71,7 +71,7 @@ public final void isOf(T... elements) {
   }
 
   public void hasContentInAnyOrder(Iterable<?> elements) {
-    verifyUsingMatcher((Matcher<? super Z>) Matchers.containsInAnyOrder(ImmutableList.copyOf(elements)));
+    verifyUsingMatcher(Matchers.containsInAnyOrder(ImmutableList.copyOf(elements).toArray()));
```

Comment:
`Iterables.toArray` may avoid a second copy.

Reply:
@Stephan202 thnx for review, fixed